services/github/github-issue-detail.spec.js

```diff
@@ -0,0 +1,84 @@
+'use strict'
+
+const { expect } = require('chai')
+const { test, given } = require('sazerac')
+const { InvalidResponse } = require('..')
+const GithubIssueDetail = require('./github-issue-detail.service')
+const { stateColor } = require('./github-helpers')
+
```

Comment:
Will happily add some other unit tests in here to validate the `render` and `transform` functions once there's no objections to the approach I took in the service implementation